source/docs/getting-started/index.html.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Introduction"
+---
+
+##### Introduction
+
+The best way to understand what kitchen does is to see it in action so we're going to use it to help us write a simple Chef cookbook. This cookbook will be complete with tests that verify the cookbook does what it's supposed to do. Kitchen comes at the process of software development with an approach that embraces the idea that writing the tests *first*, watching them fail, and then writing the code to make them pass is a great way to go.  Also known as *red, green, refactor*, it's a great way to write quality software.  By the end of this guide, you'll be familiar with this approach.
+
+For the sake of consistency through the guide, we'll be referring to Test Kitchen as just "kitchen".
```

Comment:
I renamed the front page stuff at the last minute and so that note might not be as relevant as it was - the more I think about it, the less reason I feel we should even call it "Test Kitchen" any longer.